Packs/Exabeam/Integrations/Exabeam/Exabeam_test.py

```diff
@@ -454,3 +459,56 @@ def test_validate_authentication_params(mocker, args, expected_results):
                is_fetch=args['is_fetch'])
 
     assert str(err.value) == expected_results
+
+
+@pytest.mark.parametrize(
+    'params, expected_incidents, expected_last_run',
+    [
+        (
+            {'max_fetch': 3, 'incident_type': 'generic,abnormalAuth', 'status': 'new', 'priority': 'medium', 'look_back': 4},
+            EXPECTED_INCIDENTS_FOR_LOOK_BACK,
+            EXPECTED_LAST_RUN_FOR_LOOK_BACK,
+        )
+    ]
+)
+@freeze_time(datetime(2022, 12, 22, 10, 0, 5, tzinfo=pytz.timezone('UTC')))
+def test_fetch_incidents_with_look_back(mocker, params, expected_incidents, expected_last_run):
+    """
+        Given:
+            - A last run with an incident.
+        When:
+            - executing fetch function with look back.
+        Then:
+            - Ensure that the last run time is set correctly.
+            - Check that the query with look-back is set correctly.
+            - Ensure that incidents are correctly filtered.
+    """
+    mocker.patch.object(Client, '_login', return_value=None)
+    client = Client(base_url='https://example.com', username='test_user', password='1234', verify=False, proxy=False,
```

Comment:
let's break into multiple lines of the Client assignment in order to match pep8 (https://peps.python.org/pep-0008/#introduction) guide.